src/store/state/state-model.ts

```diff
@@ -152,6 +153,9 @@ export const stateModel: StateModelIF = {
     approvalType: null,
     approvalTypeValid: true,
     businessNameValid: false,
+    courtOrder: {
+      fileNumber: null
+    },
```

Comment:
Please verify that there is conflict between this court order and the one in the newAlteration store object.

Reply:
Verified, no conflict.  We're using different getters and setters for the filings.